components/admin/requests/payment-information/EditModal.tsx

```diff
@@ -22,79 +24,16 @@ type EditPaymentDetailsModalProps = {
 
 export default function EditPaymentDetailsModal({
   children,
-  paymentInformation: currentPaymentInformation,
+  paymentInformation,
   onSave,
 }: EditPaymentDetailsModalProps) {
   const { isOpen, onOpen, onClose } = useDisclosure();
-  const [paymentInformation, setPaymentInformation] = useState<PaymentInformationFormData>(
-    currentPaymentInformation || {
-      paymentMethod: undefined,
-      donationAmount: null,
-      shippingAddressSameAsHomeAddress: false,
-      shippingFullName: '',
-      shippingAddressLine1: '',
-      shippingAddressLine2: '',
-      shippingCity: '',
-      shippingProvince: undefined,
-      shippingPostalCode: '',
-      billingAddressSameAsHomeAddress: false,
-      billingFullName: '',
-      billingAddressLine1: '',
-      billingAddressLine2: '',
-      billingCity: '',
-      billingProvince: undefined,
-      billingPostalCode: '',
-    }
-  );
-
-  useEffect(() => {
-    setPaymentInformation(currentPaymentInformation);
-  }, [currentPaymentInformation, isOpen]);
 
-  const handleSubmit = (event: SyntheticEvent) => {
-    event.preventDefault();
-    // TODO: Add error handling to each field as follows (post-mvp)
-    // if (!hideShippingInfo) {
-    //   if (!shippingFullName.length) {
-    //     setShippingFullNameInputError("Please enter the recipient's full name.");
-    //   }
-    // }
-    const {
-      paymentMethod,
-      donationAmount,
-      shippingAddressSameAsHomeAddress,
-      shippingFullName,
-      shippingAddressLine1,
-      shippingAddressLine2,
-      shippingCity,
-      shippingProvince,
-      shippingPostalCode,
-      billingAddressSameAsHomeAddress,
-      billingFullName,
-      billingAddressLine1,
-      billingAddressLine2,
-      billingCity,
-      billingProvince,
-      billingPostalCode,
-    } = paymentInformation;
-    onSave({
-      ...(paymentMethod && { paymentMethod }),
-      ...(donationAmount !== undefined && { donationAmount }),
-      shippingAddressSameAsHomeAddress,
-      shippingFullName,
-      shippingAddressLine1,
-      shippingAddressLine2,
-      shippingCity,
-      shippingProvince,
-      shippingPostalCode,
-      billingAddressSameAsHomeAddress,
-      billingFullName,
-      billingAddressLine1,
-      billingAddressLine2,
-      billingCity,
-      billingProvince,
-      billingPostalCode,
-    });
+  const handleSubmit = (values: { paymentInformation: PaymentInformationFormData }) => {
+    // TODO FIX: Card is passing in processingFee in object which is breaking request and needs to be removed
+    const { processingFee, ...pickedValues } = values.paymentInformation;
```

Comment:
I'll look into this tonight/tomorrow. Didn't want this minor issue to delay reviews so I still requested reviews

Reply:
I resolved this by manually picking all of the values which is how it was being done before